modules/apps/portal-scheduler/portal-scheduler-quartz/src/main/java/com/liferay/portal/scheduler/quartz/internal/QuartzTrigger.java

```diff
@@ -21,7 +21,7 @@
 import org.quartz.JobKey;
 
 /**
- * @author Tina Tian
+ * @author Tina Tian Tiann
```

Comment:
Tina Tian Tiann ? Is it right?

Reply:
This is a dummy change, will not be a part of the PR, when it will be forwarded.